.bashrc

```diff
@@ -0,0 +1,2 @@
+export PATH=“[PATH_TO_FLUTTER_SDK_DIRECTORY]/flutter/bin:$PATH”
```

Comment:
This file should not be in the project. please remove it.

Reply:
Done.